src/mainwindow.cpp

```diff
@@ -203,20 +205,16 @@ void MainWindow::load_config(QString filename) {
 
 		buildFilename();
 
-		// Check the wild-card-replaced filename to see if it exists already.
-		// If it does then increment the exp number.
-		// We only do this on settings-load because manual spin changes might indicate purposeful
-		// overwriting.
-		QString recFilename = QDir::cleanPath(ui->lineEdit_template->text());
-		// Spin Number
-		if (recFilename.contains(counterPlaceholder())) {
-			for (int i = 1; i < 1001; i++) {
-				ui->spin_counter->setValue(i);
-				if (!QFileInfo::exists(replaceFilename(recFilename))) break;
-			}
+		// Added by @Doug1983 to increase config file functionality:
+		// Append BIDS modalities to the default list. Need to be called after buildFilename
+		// since the default values are added there.
+		QStringList bidsModalities;
+		if (pt.contains("BidsModalities")) {
```

Comment:
The modalities below should be the default settings

Reply:
Sorry can you clarify what you mean here?

Reply:
`QSettings::value` takes a default value as second parameter in case the config file doesn't have the key, so the entire block https://github.com/labstreaminglayer/App-LabRecorder/blob/18ccd54a243c4dd3205f61c77d359bb99892b861/src/mainwindow.cpp#L211-L215 could be shortened to `pt.value("BidsModalities", QStringList())` or even with the default values from https://github.com/labstreaminglayer/App-LabRecorder/blob/18ccd54a243c4dd3205f61c77d359bb99892b861/src/mainwindow.cpp#L445-L449 `pt.value.("BidsModalities", QStringList({"eeg", "ieeg", "meg", "beh"}))`